node/forest_libp2p/src/service.rs

```diff
@@ -188,6 +189,22 @@ fn get_keypair(log: &Logger) -> Keypair {
                 Err(e) => {
                     info!(log, "Could not decode networking keystore!");
                     trace!(log, "Error {:?}", e);
+                    // Get current number of old files
+                    let file_count =
+                        count_files(path_to_keystore[0..path_to_keystore.len() - 8].to_string());
+                    match file_count {
+                        Err(e) => {
+                            info!(log, "Error {:?}", &e);
+                        }
+                        Ok(v) => {
+                            fs::rename(
+                                path_to_keystore.clone(),
+                                path_to_keystore.clone() + &format!(".old({:})", v),
+                            )
+                            .expect("label the file as old key");
```

Comment:
don't panic if this fails, should be recoverable

Reply:
ok